float64[4] speed_is
float64[4] steering_is
float64[3] distance_front
float64[3] distance_back
int8 error_id
string  error_message
time time_frame
time delta_tx2_rio
